lib/stove.rb

```diff
@@ -1,5 +1,5 @@
-require 'logify'
 require 'pathname'
+require 'chef/log'
```

Comment:
The whole point of stove is that it doesn't dep on chef. We need to use mixlib-log directly.